rtb-house/rtb-house-htb.js

```diff
@@ -420,8 +478,8 @@ function RTBHouseHtb(configs) {
             },
             features: {
                 demandExpiry: {
-                    enabled: true,
-                    value: 55
+                    enabled: false,
```

Comment:
Wartości mniejsze niż ~200ms powodowały faila w testach (na wyśledzenie tego mnóstwo czasu, bo nie szukałem w tym miejscu od początku).
Wszystkie inne adaptery mają tutaj 'false'. Pytanie, jak bardzo nas to potencjalnie boli.

Reply:
Linkniesz do doku? 
Moj "feeling" tego feata jest taki, ze tu mamy powiedziec ile [sekund? milisekund? Czegos innego?] nasz bid ma byc wazny. Po tym czasie powinien zostac inwalidowany
Jezeli to to - to chcielibysmy tu dac 55 sekund

Reply:
Aaa, jak pełnych sekund - to 55s nie powinno przeszkadzać (na pewno nie w testach), przestawię na 55000 w takim razie.
Zafiksowałem się, że to milisekundy.
doku: https://github.com/indexexchange/ix-library-adapters szukaj "demandExpiry"